components/multimedia/libav/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Friedrich Kink
+#
+
+OPENSSL_VERSION= 3.1
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		libav
+COMPONENT_VERSION=	12.3
+COMPONENT_FMRI=		multimedia/libav
+COMPONENT_SUMMARY=	A set of portable, functional and high-performance libraries for \
+	dealing with multimedia formats of all sorts
+COMPONENT_CLASSIFICATION=System/Multimedia Libraries
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	sha256:115b659022dd387f662e26fbc5bc0cc14ec18daa100003ffd34f4da0479b272e
+COMPONENT_PROJECT_URL=	https://libav.org
+COMPONENT_ARCHIVE_URL=	https://libav.org/releases/$(COMPONENT_ARCHIVE)
```

Comment:
DNS lookup for libav.org fails:
*** Can't find libav.org: No answer

Thus, no source file can be downloaded.